src/main.rs

```diff
@@ -1,14 +1,23 @@
 use anyhow::Result;
+use tracing_subscriber::{fmt, prelude::*, EnvFilter, Registry};
+use tracing_texray::TeXRayLayer;
 
 fn main() -> Result<()> {
     // this handle should be held until the end of the program,
     // do not replace by let _ = ...
     let _metrics_handle = lurk_metrics::MetricsSink::init();
-    pretty_env_logger::init();
+
+    let subscriber = Registry::default()
+        .with(fmt::layer().pretty())
+        .with(EnvFilter::from_default_env())
+        .with(TeXRayLayer::new()); // note: we don't `tracing_texray::examine` anything
```

Comment:
Yep, so this comment ("// note: we don't `tracing_texray::examine` anything **yet*") makes total sense here.